src/index.html

```diff
@@ -10,23 +10,49 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      data-qa="big"
+      class="search search--big"
+    >
+      <label
+        class="search__label search__label--big"
+        for="search__input--big"
+      >
+        <input
+          class="search__input search__input--big"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+          id="search__input--big"
+        />
+      </label>
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      data-qa="small"
+      class="search search--small"
```

Comment:
Add empty lines between multiline sibling blocks of HTML. There should be an empty line between the closing tag of the first form and the opening tag of the second form to add some 'air' and simplify reading.